locales/en/homepage.json

```
@@ -36,6 +36,9 @@
   "reuses.tosdr.author": "ToS;DR",
   "reuses.tosdr.subtitle": "Lets users collaboratively rate how respectful terms of services are of their rights.",
   "reuses.tosdr.title": "Terms of Service; Didn’t Read",
+  "reuses.report-lab-lex.author": "Lab-LEX",
```

I believe this institutional affiliation is more credible 🙂 
```suggestion
  "reuses.report-lab-lex.author": "Lab-LEX, Université de Brest",
```